--- /home/chilbolton_software/universal_radar_code/RNC/src/RNC_NetCDF.c	2010-10-04 15:14:06.000000000 +0000
+++ ./RNC_NetCDF.c	2011-06-28 15:17:05.000000000 +0000
@@ -27,7 +27,7 @@
 #include <math.h>
 
 #include <RNC.h>
-#include "/home/chilbolton_software/universal_radar_code/RSP/include/RSP.h"
+#include "/root/universal_radar_code/RSP/include/RSP.h"
 
 void RNC_OpenNetcdfFile( URC_ScanStruct *args, int radar, int *ncid )
 {
